apps/auth0/src/types.d.ts

```diff
@@ -24,8 +24,8 @@ export type DeepPartial<T> = {
 
 type Auth0Secrets = {
   secrets: {
-    PR_APP_DOMAIN: string; // hub.asap.science / gp2.asap.science
-    ASAP_API_URL: string; // https://api-dev.hub.asap.science / https://api-dev.gp2.asap.science
+    BASE_PR_APP_DOMAIN: string; // hub.asap.science / gp2.asap.science
+    API_URL: string; // https://api-dev.hub.asap.science / https://api-dev.gp2.asap.science
```

Comment:
why are these secrets?

Reply:
Because Auth0 doesn't have a concept of Environment Variables. I guess we could inline these variables when we compile but seems outside the scope of this ticket